### PR TITLE
Switch to Go modules

### PR DESCRIPTION
**Pull request by chizhg**
**What this PR does, why we need it**:
Switch test-infra to use Go modules

### CODE REVIEW

File: hack/tools.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tools
+
+import (
+    _ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
```
Comment (mattmoor) [inline hack/tools.go]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
```

Comment (mattmoor) [inline hack/tools.go]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
```

File: hack/tools.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    _ "knative.dev/pkg/testutils/metahelper"
```
Comment (mattmoor) [inline hack/tools.go]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

Comment (mattmoor) [inline hack/tools.go:23]: Format Go code:
```suggestion
	_ "knative.dev/pkg/testutils/clustermanager/prow-cluster-operation"
	_ "knative.dev/pkg/testutils/metahelper"
```

**Comment by chaodaiG** [comment]
/lgtm
Thanks!

**Pull request merged**